### PR TITLE
Add Link to reference objects from Role/ClusterRole details view

### PR DESCRIPTION
**Pull request by yolossn**
Fixes: https://github.com/kinvolk/headlamp/issues/663

### CODE REVIEW

**Review by joaquimrocha — CHANGES_REQUESTED** [review]
Looks good. Left small comments.

File: frontend/src/components/role/BindingDetails.tsx [hunk 1]
```diff
@@ -47,7 +46,20 @@ export default function RoleBindingDetails() {
...
+                    // item can hold a reference to non kube Objects
+                    // such as user and group names, in that case
+                    // dont create a link.
+                    !item?.apiGroup ? (
+                      <Link
+                        routeName={item.kind}
+                        params={{ namespace: item.namespace, name: item.name }}
+                      >
+                        {item.name}
+                      </Link>
```
[the remark this answers is not in the document]
Comment (yolossn) [reply frontend/src/components/role/BindingDetails.tsx:59]: The item here doesn't implement the `KubeObjectInterface` so I think we won't be able to use `ResourceLink` here.

https://github.com/kinvolk/headlamp/blob/main/frontend/src/components/common/Resource/Resource.tsx#L54

**Review by ashu8912 — APPROVED** [review]
LGTM

**Comment by joaquimrocha** [comment]
I've tested this on a cluster where in one case, a RoleBinding subjects do not have a namespace set to them, and these changes have an error in such case and they expect a namespace to exist.

So I added a fixup commit to avoid that.

**Pull request merged**